Text and config changes for v2.7.

JGuetschow opened the pull request:
Only the changes to configuration and code not the data and changelog so review in github is possible.

I also changed the text for commercial customers, that's the most important part to review.

JGuetschow committed:
Changes for v2.7. Only the changes to configuration and code not the data and changelog so review in github is possible.

mikapfl commented on primap-hist/template.jinja2.html:
```diff
@@ -116,10 +116,10 @@ <h3>Dataset Download (Version {{ data["metadata"]["version"] }})</h3>
 <p>
-Climate Resource makes this data available <a class="primap-blue" href="https://creativecommons.org/licenses/by/4.0/ ">CC BY 4.0</a> licence. We also provide additional data, and data support services to clients wanting more frequent updates, additional metadata or to integrate these datasets into their workflows. Get in touch at <a class="primap-blue" href="mailto:contact@climate-resource.com">c ...
+Johannes Gütschow and Climate Resource make this data available under the <a class="primap-blue" href="https://creativecommons.org/licenses/by-nc-sa/4.0/ ">CC BY-NC-SA 4.0</a> licence. This means that the data can be freely used for non-commercial purposes. For commercial customers we offer specific commercial licenses. For licenses for commercial use by NGOs and internal use by small companies p ...
 </p>
```
```suggestion
<p>
Johannes Gütschow and Climate Resource make this data available under the <a class="primap-blue" href="https://creativecommons.org/licenses/by-nc-sa/4.0/ ">CC BY-NC-SA 4.0 licence</a>. This means that the data can be freely used for non-commercial purposes as long as you provide adequate citations and share your derived results under the same licence. For other uses and commercial customers, we offer specific commercial licenses. 
</p>
<p>
For licenses for commercial use by NGOs and internal use by small companies, please contact Johannes Gütschow (<a class="primap-blue" href="mailto:commercial-support@johannes-guetschow.de">commercial-support@johannes-guetschow.de</a>).
</p>
<p>
Larger companies, including professional services and research providers, banks, insurance, investment companies and other financial institutions, multilateral organisations, and governments, please contact Mika Pflüger at Climate Resource (<a class="primap-blue" href="mailto:primap@climate-resource.com">primap@climate-resource.com</a>). 
</p>
<p>
We also provide additional data, and data support services to clients wanting more frequent updates, additional metadata or to integrate these datasets into their workflows. Get in touch if you are interested.
</p>
<p>
Commercial use is defined as the use of the data for research or advice you have been contracted to provide, including the production of data or information you have derived from or that incorporates data or information obtained from this website for commercial purposes. This includes in the provision of data, advice or other services you have been contracted to provide, irrespective of whether you are a for-profit, or not-for-profit entity. It also includes the use of the data for investment decisions.
</p>
```

mikapfl replied:
Also added "governments and multilateral organisations" to our list because we already do that - but happy to change it / leave it out if you don't want to decide that now.

JGuetschow replied:
Makes sense and fine with me, just one question: The commercial use clause would not classify an activity as commercial if it's not contracted and the result is provided free of charge, correct? This makes sense to me, so just to see if I understood correctly.

mikapfl replied:
If it's not contracted and the result is provided free of charge under non-commercial terms as well. So, it would not be legal for some company A to make an analysis uncontracted and publish it such that company B can then use it commercially.

mikapfl replied:
So, the share-alike is not part of the question if a use is commercial, but to be able to use the data for free, you have to be a) uncommercial (as defined by the licence and specified by the clause) and b) if you re-publish the data or derived results, you have to do so under the same conditions. So even non-commercial users who want to e.g. re-publish derived results under a different license will need a separate license. I don't know if we need to spell that out somewhere as well?

JGuetschow replied:
Maybe we can add "For all non-commercial use cases published results have to use the CC BY-NC-SA license".

mikapfl replied:
sounds good, accepted.

JGuetschow replied:
Great, I'll push the data files later and will complete the update afterwards.

mikapfl reviewed (commented):
Thanks for updating! I suggest some wording updates and adding a small definition of commercial use, to avoid misunderstandings. Also, we found that we have a specialised contact address primap@climate-resource.com, so we would like to use that.

JGuetschow commented on primap-hist/template.jinja2.html:
```diff
@@ -116,10 +116,10 @@ <h3>Dataset Download (Version {{ data["metadata"]["version"] }})</h3>
 <p>
-Climate Resource makes this data available <a class="primap-blue" href="https://creativecommons.org/licenses/by/4.0/ ">CC BY 4.0</a> licence. We also provide additional data, and data support services to clients wanting more frequent updates, additional metadata or to integrate these datasets into their workflows. Get in touch at <a class="primap-blue" href="mailto:contact@climate-resource.com">c ...
+Johannes Gütschow and Climate Resource make this data available under the <a class="primap-blue" href="https://creativecommons.org/licenses/by-nc-sa/4.0/ ">CC BY-NC-SA 4.0</a> licence. This means that the data can be freely used for non-commercial purposes. For commercial customers we offer specific commercial licenses. For licenses for commercial use by NGOs and internal use by small companies p ...
 </p>
```
```suggestion
For all non-commercial use cases published results have to use the CC BY-NC-SA license.
</p>
```

mikapfl committed:
Update primap-hist/template.jinja2.html

Co-authored-by: Johannes Gütschow <mail@johannes-guetschow.de>

mikapfl committed:
Update primap-hist/template.jinja2.html

mikapfl commented:
If you're happy with it, probably the html needs to be re-generated? I think then it can be merged.

JGuetschow commented:
Yes, the html needs to be regenerated. I'm currently on my laptop and the data files are on my workstation, so I'll do the html regeneration together with the data upload later today.

JGuetschow committed:
Added data
Added new chnagelog
Some layout modifications to make space for the commercial use text

JGuetschow commented:
I had to change the layout a bit because the new text is quite long. Now the blue bar is below the plot, so the plot is visible without scrolling.
